packages/oas/README.md

```diff
@@ -76,13 +76,13 @@ Because this library has full TypeScript types and docblocks this README is not
 <!-- prettier-ignore-start -->
 | Method | Description |
 | :--- | :--- |
-| `#dereference()` | Dereference the current OpenAPI definition. Note that this will ignore circular references. |
```

Comment:
Changed these `#` to a `.` to better indicate that they're instance methods, not static.